english/alive/alive.md

```diff
@@ -0,0 +1 @@
+Click [here](https://www.youtube.com/watch?v=U130wnpi-C0) for the Alive trailer!
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.